File: ComplementApp.API/Dtos/RelacionContableDto.cs
namespace ComplementApp.API.Dtos
{
    public class RelacionContableDto
    {
        public int RelacionContableId { get; set; }
        public int? TipoOperacion { get; set; }
        public int? UsoContable { get; set; }
        public ValorSeleccion CuentaContable { get; set; }
        public ValorSeleccion AtributoContable { get; set; }
        public ValorSeleccion TipoGasto { get; set; }
    }
}

File: ComplementApp-SPA/src/app/_dto/relacionContableDto.ts
import { ValorSeleccion } from './valorSeleccion';

export interface RelacionContableDto {
  relacionContableId: number;
  tipoOperacion: number;
  usoContable: number;
  cuentaContable: ValorSeleccion;
  atributoContable: ValorSeleccion;
  tipoGasto: ValorSeleccion;
}
